### PR TITLE
Update daedalus to 0.11.0

### PR DESCRIPTION
**Pull request by gtklocker**
Versioning for daedalus has been a little weird, this seeming downgrade is actually an upgrade. https://daedaluswallet.io/release-notes/

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [ ] Named the cask according to the [token reference].
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked there are no [open pull requests] for the same cask.
- [ ] Checked the cask was not [already refused].
- [ ] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Review by adidalal — DISMISSED** [review]

File: Casks/daedalus.rb [hunk 1]
```diff
@@ -1,15 +1,15 @@
 cask 'daedalus' do
-  version '1.1.1.952'
-  sha256 'f5262107b30f5617018fec7bff42982484f3af3623f8169d98500dfb7d495b84'
+  version '0.11.0'
+  sha256 '6ad90da2053f06f9f0e35c2ae05a8620094b83c6a6a5401c201a1e5abcc22ae4'
 
-  # amazonaws.com/update-cardano-mainnet.iohk.io was verified as official when first introduced to the cask
-  url "https://s3-ap-southeast-1.amazonaws.com/update-cardano-mainnet.iohk.io/Daedalus-installer-#{version}.pkg"
+  # update-cardano-mainnet.iohk.io was verified as official when first introduced to the cask
```
Comment (gtklocker) [inline Casks/daedalus.rb]: Is this the right way to change this line?

File: Casks/daedalus.rb [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
   # amazonaws.com/update-cardano-mainnet.iohk.io was verified as official when first introduced to the cask
-  url "https://s3-ap-southeast-1.amazonaws.com/update-cardano-mainnet.iohk.io/Daedalus-installer-#{version}.pkg"
+  url "https://update-cardano-mainnet.iohk.io/daedalus-#{version}-cardano-sl-1.3.0-mainnet-macos-2260.pkg"
   name 'Daedalus'
   homepage 'https://daedaluswallet.io/'
 
-  pkg "Daedalus-installer-#{version}.pkg"
+  pkg "daedalus-#{version}-cardano-sl-1.3.0-mainnet-macos-2260.pkg"
 
   uninstall pkgutil: 'org.daedalus.pkg',
```
[the remark this answers is not in the document]
Comment (gtklocker) [reply Casks/daedalus.rb]: fixed

**Review by vitorgalvao — APPROVED** [review]

File: Casks/daedalus.rb (hunk 1 above)
Comment (vitorgalvao) [reply Casks/daedalus.rb]: Yes.

**Comment by gtklocker** [comment]
@vitorgalvao does this look good enough to merge now?

**Comment by vitorgalvao** [comment]
> does this look good enough to merge now?

Yes, I was just waiting for Travis to pass.

**Comment by jedahan** [comment]
I am unable to install the new version of daedalus:

```bash
 △ ~ brew cask install daedalus                                                                 
==> Satisfying dependencies
==> Downloading https://github.com/input-output-hk/daedalus/releases/download/0.11.0/daedalus-0.11
Already downloaded: /Users/micro/Library/Caches/Homebrew/downloads/27a7ef9eb401847648c60ff51bc8461b0840a01b6d83307e9cc072fb66cb8d7c--daedalus-0.11.0-cardano-sl-1.3.0-mainnet-macos-2260.pkg
==> Verifying SHA-256 checksum for Cask 'daedalus'.
==> Installing Cask daedalus
==> Running installer for daedalus; your password may be necessary.
==> Package installers may write to any location; options such as --appdir are ignored.
installer: Package name is
installer: Installing at base path /
installer: The install failed (The Installer encountered an error that caused the installation to fail. Contact the software manufacturer for assistance.)
==> Purging files for version 0.11.0,1.3.0:2260 of Cask daedalus
Error: Failure while executing; `/usr/bin/sudo -E -- env LOGNAME=micro USER=micro USERNAME=micro /usr/sbin/installer -pkg /usr/local/Caskroom/daedalus/0.11.0,1.3.0:2260/daedalus-0.11.0-cardano-sl-1.3.0-mainnet-macos-2260.pkg -target /` exited with 1. Here's the output:
installer: Package name is
installer: Installing at base path /
installer: The install failed (The Installer encountered an error that caused the installation to fail. Contact the software manufacturer for assistance.)

Follow the instructions here:
  https://github.com/Homebrew/homebrew-cask#reporting-bugs
/usr/local/Homebrew/Library/Homebrew/system_command.rb:101:in `assert_success'
/usr/local/Homebrew/Library/Homebrew/system_command.rb:46:in `run!'
/usr/local/Homebrew/Library/Homebrew/system_command.rb:23:in `run'
/usr/local/Homebrew/Library/Homebrew/system_command.rb:27:in `run!'
/usr/local/Homebrew/Library/Homebrew/cask/artifact/pkg.rb:58:in `block in run_installer'
/usr/local/Homebrew/Library/Homebrew/cask/artifact/pkg.rb:64:in `with_choices_file'
/usr/local/Homebrew/Library/Homebrew/cask/artifact/pkg.rb:50:in `run_installer'
/usr/local/Homebrew/Library/Homebrew/cask/artifact/pkg.rb:31:in `install_phase'
/usr/local/Homebrew/Library/Homebrew/cask/installer.rb:207:in `block in install_artifacts'
/usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.3.7/lib/ruby/2.3.0/set.rb:674:in `each'
/usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.3.7/lib/ruby/2.3.0/set.rb:674:in `each'
/usr/local/Homebrew/Library/Homebrew/cask/installer.rb:199:in `install_artifacts'
/usr/local/Homebrew/Library/Homebrew/cask/installer.rb:96:in `install'
/usr/local/Homebrew/Library/Homebrew/cask/cmd/install.rb:21:in `block in run'
/usr/local/Homebrew/Library/Homebrew/cask/cmd/install.rb:14:in `each'
/usr/local/Homebrew/Library/Homebrew/cask/cmd/install.rb:14:in `run'
/usr/local/Homebrew/Library/Homebrew/cask/cmd/abstract_command.rb:34:in `run'
/usr/local/Homebrew/Library/Homebrew/cask/cmd.rb:89:in `run_command'
/usr/local/Homebrew/Library/Homebrew/cask/cmd.rb:155:in `run'
/usr/local/Homebrew/Library/Homebrew/cask/cmd.rb:120:in `run'
/usr/local/Homebrew/Library/Homebrew/cmd/cask.rb:7:in `cask'
/usr/local/Homebrew/Library/Homebrew/brew.rb:89:in `<main>'
```

**Comment by adidalal** [comment]
@jedehan please open up an issue in the appropriate location, following the template.

**Pull request merged**